src/components/ui/AmountPaid/AmountPaid.tsx

```diff
@@ -0,0 +1,40 @@
+import { Currency } from '../../../api/types/Enums';
+
+const AmountPaid = ({
+  amountPaid,
+  totalAmount,
+  currency,
+}: {
+  amountPaid: number;
+  totalAmount: number;
+  currency: Currency.EUR | Currency.GBP;
+}) => {
+  const percentagePaid = amountPaid > totalAmount ? 100 : (amountPaid / totalAmount) * 100;
+  const outstandingAmount = amountPaid > totalAmount ? 0 : totalAmount - amountPaid;
+  return (
+    <>
+      <div className="flex flex-col">
+        <span
+          className={`text-[2rem] tabular-nums font-semibold leading-8 mb-4  ${
+            amountPaid === 0 ? 'text-[#73808C]' : 'text-[#00264D]'
+          }`}
+        >
+          {currency == Currency.GBP ? '£' : '€'}
+          {amountPaid === 0 ? totalAmount : amountPaid.toFixed(2)}
+        </span>
+        <div className="flex flex-row h-[0.125rem]">
+          <div className="bg-[#40BFBF] rounded-l-sm" style={{ flexGrow: `${percentagePaid}` }}></div>
+          <div className="bg-[#EDF2F7] rounded-r-sm" style={{ flexGrow: `${100 - percentagePaid}` }}></div>
+        </div>
+        {amountPaid > 0 && (
+          <span className="text-sm leading-[1.063rem] text-[#73808C] font-medium mt-2">
+            {currency == Currency.GBP ? '£' : '€'}
+            <span className="tabular-nums">{outstandingAmount.toFixed(2)}</span> outstanding
+          </span>
+        )}
+      </div>
+    </>
```

Comment:
Just noticed a few more things.

1. Fragment (`<> </>`) is not needed as component returns only one element.
2. Substitution of classes should be done by using the `classNames` modules. [Here](https://github.com/nofrixion/nofrixion.webcomponents/blob/develop/src/components/ui/CreatePaymentRequestPage/CreatePaymentRequestPage.tsx#L409-L411)'s an example.
3. All HEX colours in here are actually available through names as they were added in the tailwind.config file.

Here are a few examples of classes to change:
- From `text-[#73808C]` to `text-greyText`
- From `text-[#73808C]` to `text-defaultText`
- From `bg-[#40BFBF]` to `bg-positiveActionBackground`
- and so on...

Reply:
> Just noticed a few more things.
> 
> 1. Fragment (`<> </>`) is not needed as component returns only one element.
> 2. Substitution of classes should be done by using the `classNames` modules. [Here](https://github.com/nofrixion/nofrixion.webcomponents/blob/develop/src/components/ui/CreatePaymentRequestPage/CreatePaymentRequestPage.tsx#L409-L411)'s an example.
> 3. All HEX colours in here are actually available through names as they were added in the tailwind.config file.
> 
> Here are a few examples of classes to change:
> 
> * From `text-[#73808C]` to `text-greyText`
> * From `text-[#73808C]` to `text-defaultText`
> * From `bg-[#40BFBF]` to `bg-positiveActionBackground`
> * and so on...

Oh thanks @Publicker. Will make these changes.